nilearn/plotting/js_plotting_utils.py

```diff
@@ -76,8 +76,8 @@ def colorscale(cmap, values, threshold=None, symmetric_cmap=True,
         warnings.warn('you have specified symmetric_cmap=False '
                       'but the map contains negative values; '
                       'setting symmetric_cmap to True',
-                      stacklevel=3)
-        symmetric_cmap = True
+                      stacklevel=4)
+        # symmetric_cmap = True
```

Comment:
no `symmetric_cbar` handling in the code for plotly so trying to rely on `symmetric_cmap`

silencing the part that forces a `symmetric_cmap=true` just for testing